uxarray/grid.py

```diff
@@ -225,3 +227,34 @@ def __init_ds_var_names__(self):
             "nMesh2_face": "nMesh2_face",
             "nMaxMesh2_face_nodes": "nMaxMesh2_face_nodes"
         }
+
+    def __init_ds_var_methods__(self):
+        """Initialize variables for directly accessing Coordinate and
+        Data variables through ugrid conventions
+
+        Examples
+        ----------
+        Initialize some Grid Object
+        >>> grid = ux.open_dataset(file_path)
+
+        Return Xarray Data Array
+        >>> x = grid.Mesh2_node_x      
+
+        Return Numpy Array
+        >>> x = grid.Mesh2_node_x.values
+        """
+    
+        # For Testing (caused other tests to fail)
+        if self.ds is None:
+            return
+
+        # Swap Keys & Values (Accessed with UGRID convention) UGRID -> Orig
+        ds_ugrid_var_names = dict((k, v) for k,v in self.ds_var_names.items())
+      
+        for key, value in ds_ugrid_var_names.items():
```

Comment:
I like this implementation much overall (I was personally thinking of writing separate methods instead, but this seems more compact). 

The only concern of mine is that depending on the existence of variables and coords, the user will see some attributes showing up in Grid for some datasets and disappearing for others. I am not sure how this behavior will feel to the user. Any thoughts? @anissa111 @dcherian @rajeeja @michaelavs

Reply:
My thought process was that this approach would only create attributes for variables that are present in the dataset and dictionary. For example, if our dataset doesn't include a `Mesh2_node_z` variable, it wouldn't make sense to add an attribute for it. 

As long as there exists an entry in the `ds_var_names` dictionary that corresponds to some variable in the coordinates or data vars of our dataset, an attribute will be created allowing us to access it with it's UGRID standardized name. For variables that are not covered by UGRID, I didn't add that functionality, but can easily do that if we believe it would be beneficial.

Reply:
This makes sense to me, and we can go with your approach if there is no objection from others.